src/symbolic/symbolic_choice.ml

```diff
@@ -13,9 +13,9 @@ module Prio = struct
   (*
       Currently there is no real notion of priority. Future extensions adding it will ho here.
     *)
-  type t = Default
+  type t = Random
 
-  let default = Default
+  let random = Random  let to_int = function Random -> Random.int 10000000
```

Comment:
Something is weird with the formatting. Could you try running `dune fmt` and make a new commit?